.editorconfig

```diff
@@ -7,3 +7,6 @@ end_of_line = lf
 charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
+
+[*.yml]
+indent_size = 2
```

Comment:
Why this exception for YML files?
Shouldn't we be consistent with what we agreed (4 spaces)?